### PR TITLE
other: bump up v0.18.0

### PR DESCRIPTION
**Pull request by rebel-jiwoopark**
### 🚀 Summary of Changes

other: bump up v0.18.0

---

### 📌 Related Issues / Tickets

* Resolves https://github.com/rebellions-sw/vllm-rbln-internal/issues/8
* Related to #

---

### ✅ Type of Change

* [ ] 🚀 Release (`release`)
* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [x] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

**Review by rebel-jonghewk — APPROVED** [review]

**Comment by rebel-seinpark** [comment]
please check the lint

**Comment by rebel-jiwoopark** [comment]
> please check the lint

Thank you. I fixed the lint errors.

File: pyproject.toml [hunk 1]
```diff
@@ -38,11 +38,11 @@ dependencies = [
     "pyyaml",
     "datasets",
     "qwen_vl_utils",
-    "vllm==0.13.0+cpu",
+    "vllm==0.18.0+cpu",
     "torch",
     "torchvision",
     "torchaudio",
-    "optimum-rbln>=0.10.2",
+    "optimum-rbln @ git+https://github.com/RBLN-SW/optimum-rbln.git@upgrade-torch",
```
Comment (rebel-eunji) [inline pyproject.toml]: @rebel-seinpark generated a PR of upgrade-torch branch.
After the PR is merged into dev branch, a new tag will be here.
Thanks a lot!

**Review by rebel-eunji — APPROVED** [review]
LGTM. Before merging, please double-check that the CI using the vllm-rbln dev branch is ready and ensure that the results remain consistent after the merge.

File: pyproject.toml (hunk 1 above)
Comment (rebel-seinpark) [reply pyproject.toml]: Optimum-rbln has testing with the upgrade-torch branch. If the branch is merged i will notice this PR.

Comment (rebel-seinpark) [inline pyproject.toml]:
```suggestion
    "optimum-rbln==0.10.3a1",
```

Comment (rebel-seinpark) [reply pyproject.toml]: optimum-rbln has been tagged @rebel-jiwoopark

Comment (rebel-seinpark) [reply pyproject.toml]: I updated with this [PR](https://github.com/RBLN-SW/vllm-rbln/pull/519)
